spec/writeTimeoutError.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars */
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const fs = require('fs');
+const nock = require('nock');
+const sinon = require('sinon');
+const logger = require('@elastic.io/component-logger')();
+
+chai.use(chaiAsPromised);
+const { expect } = require('chai');
+
+if (fs.existsSync('.env')) {
+  // eslint-disable-next-line global-require
+  require('dotenv').config();
+} else {
+  process.env.ELASTICIO_API_USERNAME = 'name';
+  process.env.ELASTICIO_API_KEY = 'key';
+}
+
+const write = require('../lib/actions/write.js');
+const writeFromJson = require('../lib/actions/writeFromJson.js');
+
+// eslint-disable-next-line func-names
+describe('CSV Write Timeout', function () {
+  this.timeout(180000);
```

Comment:
can you decrease timeout for test? Or it is necessary?

Reply:
It is necessary (actually I need like 1min, but let it be)